tests/models/test_fluid_mass_balance.py

```diff
@@ -269,95 +278,134 @@ def test_tested_vs_testable_methods_single_phase_flow(
-                        0.00000000e00,
-                        2.01606417e-06,
-                        2.01606417e-06,
-                        2.01606417e-06,
-                        2.01606417e-06,
-                        4.03212834e-06,
+                        0.0,
+                        0.0,
+                        0.0,
+                        0.0,
+                        2.01066509e-06,
+                        2.01066509e-06,
+                        2.01066509e-06,
+                        2.01066509e-06,
+                        4.02133017e-06,
                     ]
                 ]
             ),
             None,
         ),
-        ("interface_darcy_flux_equation", 5e9, None),
-        ("interface_fluid_flux", 1.00803209e-06, None),
-        ("interface_flux_equation", 5e9, None),
+        ("interface_darcy_flux_equation", 5.00001e-07, None),
+        ("interface_fluid_flux", 1.00533254e-06, None),
+        ("interface_flux_equation", 5.00001e-07, None),
         ("interface_vector_source_darcy_flux", 0, None),
         (
             "mass_balance_equation",
             np.array(
                 [
-                    1.40589657e-02,
-                    1.40589689e-02,
-                    1.40589689e-02,
-                    1.40589721e-02,
-                    2.80122961e-04,
-                    2.80123025e-04,
-                    2.80122961e-04,
-                    2.80123025e-04,
-                    1.59033151e-06,
+                    3.01159202e-02,
+                    3.01177451e-02,
+                    3.01177451e-02,
+                    3.01195701e-02,
+                    5.88240034e-05,
+                    5.88276534e-05,
+                    5.88240034e-05,
+                    5.88276534e-05,
+                    -3.90246847e-06,
                 ]
             ),
             None,
         ),
-        ("mobility", 1 / 0.001, None),
-        # Combination of mobility and fluid density = rho/mu
-        # = rho_ref * exp(c_f * (p - p_ref)) / mu = 1000 * exp(4e-10 * 2e7) /  0.001
-        ("mobility_rho", 1008032.0855042734, None),
-        ("normal_permeability", 1.0, None),
-        ("permeability", 1e-20, None),
-        ("porosity", 7e-3, None),
+        # Mobility = 1 / mu
+        ("mobility", 1.0 / (water_values["viscosity"]), None),
+        # Combination of mobility and fluid density = rho / mu
+        # Mobility_rho = rho_ref * exp(c_f * (p - p_ref)) / mu
+        (
+            "mobility_rho",
+            water_values["density"]
+            * np.exp(water_values["compressibility"] * 200 * pp.BAR)
+            / water_values["viscosity"],
+            None,
+        ),
+        ("normal_permeability", granite_values["normal_permeability"], None),
+        ("permeability", granite_values["permeability"], None),
+        ("porosity", granite_values["porosity"], None),
         ("pressure", 200 * pp.BAR, None),
         # pressure_exponential = exp(c_f * (p - p_ref))
```

Comment:
Should this be removed?

Reply:
Do you mean because it doesn't test everything except that the operators actually return the solid values? I think it doesn't hurt to have it in there, but you know the models better than me, so I will follow your suggestion. Would be the same case for fluid_viscosity down below.